src/Xamarin.Android.Tools/Sdks/MonoDroidSdkBase.cs

```diff
@@ -108,7 +108,8 @@ protected static bool ValidateRuntime (string loc)
 			return !string.IsNullOrWhiteSpace (loc) &&
 				(File.Exists (Path.Combine (loc, DebugRuntime)) ||    // Normal/expected
 				 File.Exists (Path.Combine (loc, ClassParseExe)) ||    // Normal/expected
-					File.Exists (Path.Combine (loc, "Ionic.Zip.dll")));  // Wrench builds
+					File.Exists (Path.Combine (loc, "Ionic.Zip.dll")) || // Wrench builds
```

Comment:
We should remove this check, as we shouldn't even be using `Ionic.Zip.dll` anymore!